packages/terra-navigation-layout/src/NavigationLayoutContent.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Redirect,
+} from 'react-router-dom';
+import classNames from 'classnames/bind';
+
+import AppDelegate from 'terra-app-delegate';
+import RoutingStack from './routing/RoutingStack';
+import { supportedComponentBreakpoints, routeConfigPropType } from './configurationPropTypes';
+
+import styles from './NavigationLayoutContent.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * The AppDelegate instance that will be propagated to the RoutingStack and its contents.
+   */
+  app: AppDelegate.propType,
+  /**
+   * The routing configuration from which Routes will be generated.
+   */
+  routeConfig: routeConfigPropType,
+  /**
+   * The current responsive size.
+   */
+  navigationLayoutSize: PropTypes.oneOf(supportedComponentBreakpoints),
+  /**
+   * The String path to redirect to should the routes specified by the routeConfig all fail to match.
+   */
+  redirect: PropTypes.string,
```

Comment:
most API's (mostly the authentication ones) I see usually have this as redirectURI, and I think that sounds better as well, as redirect sounds more like a verb than a noun

Reply:
I agree, I'll update this to `redirectPath` to keep the language consistent with other components.